airframe-sql/src/main/scala/wvlet/airframe/sql/model/ResolvedPlan.scala

```diff
@@ -24,11 +24,20 @@ import wvlet.airframe.sql.model.LogicalPlan.Relation
   * @param columns
   *   projectec columns
   */
-case class TableScan(table: Catalog.Table, columns: Seq[Catalog.TableColumn]) extends Relation with LeafPlan {
+case class TableScan(table: Catalog.Table, columns: Seq[Catalog.TableColumn], nodeLocation: Option[NodeLocation])
+    extends Relation
+    with LeafPlan {
   override def inputAttributes: Seq[Attribute] = Seq.empty
   override def outputAttributes: Seq[Attribute] = {
     columns.map { col =>
-      ResolvedAttribute(col.name, col.dataType, None, Some(table), Some(col))
+      ResolvedAttribute(
+        col.name,
+        col.dataType,
+        None,
+        Some(table),
+        Some(col),
+        None // ResolvedAttribute always has no NodeLocation
+      )
```

Comment:
`ResolvedAttribute` always has no `NodeLocation`

Reply:
It's ok for now. TypeResolve needs to propagate NodeLocation wherever possible